packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -325,6 +326,19 @@ const styledTemplate = (opts: StyledTemplateOpts, meta: Metadata): t.Node => {
       })
     : t.identifier('style');
 
+  const unconditionalClassNames: string[] = [];
+  let logicalClassNames = '';
```

Comment:
I would make this consistent have have `logicalClassNames` also be an array. Or, have them both be a string since `+=` is faster, and then just trim the trailing space in `logicalClassNames`

Reply:
- [ ] have string for `unconditionalClassNames` so that it is faster and consistent with `logicalClassNames`